pkg/controller/sparkapplication/controller.go

```diff
@@ -752,6 +760,9 @@ func (c *Controller) submitSparkApplication(app *v1beta2.SparkApplication) *v1be
 		return app
 	}
 
+	if c.metrics != nil {
+		c.metrics.sparkSubmitLatency.Observe(float64(latency / time.Second))
```

Comment:
does the .Sub() above return a duration, in nanoseconds? Do we need to divide by nanoseconds in a second here? not milliseconds